frontend/src/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionDetailsView.tsx

```diff
@@ -141,51 +141,51 @@ const ModelVersionDetailsView: React.FC<ModelVersionDetailsViewProps> = ({
         ) : (
           <>
             <DescriptionList>
-              {storageFields && (
+              {storageFields?.s3Fields && (
                 <>
                   <DashboardDescriptionListGroup
                     title="Endpoint"
-                    isEmpty={modelArtifacts.size === 0 || !storageFields.endpoint}
+                    isEmpty={modelArtifacts.size === 0 || !storageFields.s3Fields.endpoint}
```

Comment:
Now that `storageFields.s3Fields` will only be defined if there's an artifact with a URI starting with `s3:`, I think the `modelArtifacts.size === 0` case on all of these `isEmpty` props is unnecessary.